public/app/features/trails/ActionTabs/BreakdownScene.tsx

```diff
@@ -201,26 +260,33 @@ export function buildAllLayout(options: Array<SelectableValue<string>>, queryDef
     const expr = queryDef.queries[0].expr.replaceAll(VAR_GROUP_BY_EXP, String(option.value));
     const unit = queryDef.unit;
 
+    const vizPanel = PanelBuilders.timeseries()
+      .setTitle(option.label!)
+      .setData(
+        new SceneQueryRunner({
+          maxDataPoints: 300,
+          datasource: trailDS,
+          queries: [
+            {
+              refId: 'A',
+              expr: expr,
+              legendFormat: `{{${option.label}}}`,
+            },
+          ],
+        })
+      )
+      .setHeaderActions(new SelectLabelAction({ labelName: String(option.value) }))
+      .setUnit(unit)
```

Comment:
This is extrapolated out into a `vizPanel` reference so we can change the options.
Due to the ambiguous type, we must us `onOptionsChange` in an activation handler.